internal/factsengine/gatherers/hostsfile_test.go

```diff
@@ -65,6 +70,36 @@ func (suite *HostsFileTestSuite) TestHostsFileBasic() {
 			}},
 			CheckID: "check3",
 		},
+		{
+			Name: "hosts_all",
+			Value: &entities.FactValueMap{
```

Comment:
How does this get transformed in our protobuf contracts?

From an elixir point of view I imagine wanda would get something like this

```elixir
%{
  "value" => %{
    "node1" => %{
      "value" => [
        "123.456.789",
        "1.1.1.1"
      ]
    }
  }
}
```
which in the context of an Evaluation becomes
```elixir
context = %{
  "facts" => %{
    "available_host_names" => %{
      "value" => %{
        "node1" => %{
          "value" => [
            "123.456.789",
            "1.1.1.1"
          ]
        }
      }
    }
  }
}
```

requiring to access to the fact via `.value`

```rust
["node1"].all(|cluster_node| cluster_node in facts.available_host_names.value)
```

Reply:
I think so, but I invoke @fabriziosestito :smile: